### PR TITLE
Small fixes - part 2 :)

### PR DESCRIPTION
**Pull request by wtrocki**
Set of minor fixes.
UI improvement:

screen shot 2019-02-21 at 4 13 20 pm

Removing profile data after pressing logout/login

### CODE REVIEW

File: src/app/pages/home/home.page.html [hunk 1]
```diff
@@ -13,23 +13,8 @@
   <ion-card class="welcome-card">
     <ion-img class="logo" src="assets/aerogear.png"></ion-img>
     <ion-card-header>
-      <ion-card-title text-center>AeroGear Mobile Services Showcase Application</ion-card-title>
+      <ion-card-title text-center>AeroGear Mobile Services Showcase</ion-card-title>
```
Comment (wei-lee) [inline src/app/pages/home/home.page.html]: not sure if we really need this title. Will it look better if we remove this completely?

**Comment by wtrocki** [comment]
screen shot 2019-02-21 at 4 27 42 pm

With text removed. IMHO even better.

**Comment by wei-lee** [comment]
@wtrocki yeah, I think so. one very small thing: can we remove the boarder around the logo?

**Comment by wtrocki** [comment]
screen shot 2019-02-21 at 4 37 58 pm

**Comment by wei-lee** [comment]
:shipit:

**Pull request merged**